### ISSUE TITLE
Module package installation wwwroot issue in azure

### ISSUE THREAD

**Issue by tfindlow** [oqtane/oqtane.framework#844]
The module package installer in OqtaneServer.Infrastructure\InstallationManager.cs does a text replace on the "\wwwroot" value in the IWebHostEnvironment.WebRootPath.  Unfortunately the \wwwroot value appears twice in the azure hosting environment WebRootPath.  

ContentRootPath = D:\home\site\wwwroot 
WebRootPath = D:\home\site\wwwroot\wwwroot

It's probably better to replace the leading "wwwroot\" in the entry.FullName

My hack... seems to work ;-)

`filename = Path.Combine(webRootPath, Utilities.PathCombine(entry.FullName.Substring(8).Split('/')));
`

```
>>> InstallationManager.cs

                            foreach (ZipArchiveEntry entry in archive.Entries)
                            {
                                string foldername = Path.GetDirectoryName(entry.FullName).Split(Path.DirectorySeparatorChar)[0];
                                string filename = Path.GetFileName(entry.FullName);

                                switch (foldername)
                                {
                                    case "lib":
                                        filename = Path.Combine(binFolder, filename);
                                        ExtractFile(entry, filename);
                                        assets.Add(filename);
                                        break;
                                    case "wwwroot":
                                        //****** PROBLEM HERE
                                        filename = Path.Combine(webRootPath.Replace(Path.DirectorySeparatorChar + "wwwroot", ""), Utilities.PathCombine(entry.FullName.Split('/')));
                                        ExtractFile(entry, filename);
                                        assets.Add(filename);
                                        break;
                                    case "runtimes":
                                        var destSubFolder = Path.GetDirectoryName(entry.FullName);
                                        filename = Path.Combine(binFolder, destSubFolder, filename);
                                        ExtractFile(entry, filename);
                                        assets.Add(filename);
                                        break;
                                }
                            }

```

**Comment by sbwalker** [comment]
Good catch! This definitely needs to be fixed as I expect many people will want to use Oqtane on Azure.

**Comment by sbwalker** [comment]
Fixed in #846
